tests/irdl/test_declarative_assembly_format.py

```diff
@@ -3155,3 +3156,155 @@ def test_multiple_operand_extraction_fails():
         "Possible values are: {i32, index}",
     ):
         parser.parse_operation()
+
+
+################################################################################
+#                                  IntAttr                                     #
+################################################################################
+
+
+@pytest.mark.parametrize(
+    "program",
+    ["%0 = test.int_attr_extract 1", "%0, %1 = test.int_attr_extract 2"],
+)
+def test_int_attr_extraction(program: str):
+    @irdl_op_definition
+    class IntAttrExtractOp(IRDLOperation):
```

Comment:
Mind pulling this and one below out and use a single definition? Fewer lines of code to eyeball over.